scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -157,14 +158,17 @@ function Install-SpartanAgent {
 }
 
 function Install-AdminRouterAgent {
-    & "$SCRIPTS_DIR\scripts\adminrouter-agent-setup.ps1" -AgentPrivateIP $AgentPrivateIP
+    & "$SCRIPTS_DIR\scripts\adminrouter-agent-setup.ps1" -AgentPrivateIP $AgentPrivateIP -AdminRouterWindowsBinariesURL $ADMINROUTER_BINARIES_URL
     if($LASTEXITCODE) {
         Throw "Failed to setup the DCOS AdminRouter Windows agent"
     }
 }
 
 function Install-DiagnosticsAgent {
-    & "$SCRIPTS_DIR\scripts\diagnostics-agent-setup.ps1" -DiagnosticsWindowsBinariesURL $DIAGNOSTICS_BINARIES_URL
+    Param(
+        [bool]$IncludeMetricsToSericeList
```

Comment:
This shouldn't included the changes with disabling `Metrics`. That's a separate PR.

Reply:
I will make sure this was not included once the other PR was merged.